lib/images/explore/exclamation_circle.dart

```diff
@@ -0,0 +1,113 @@
+    final Path path_0 = Path();
+    path_0.moveTo(size.width * 0.5000000, size.height * 0.9375000);
+    path_0.cubicTo(size.width * 0.3839682, size.height * 0.9375000, size.width * 0.2726886, size.height * 0.8914068,
+        size.width * 0.1906407, size.height * 0.8093591);
+    path_0.cubicTo(size.width * 0.1085936, size.height * 0.7273114, size.width * 0.06250000, size.height * 0.6160318,
+        size.width * 0.06250000, size.height * 0.5000000);
+    path_0.cubicTo(size.width * 0.06250000, size.height * 0.3839682, size.width * 0.1085936, size.height * 0.2726886,
+        size.width * 0.1906407, size.height * 0.1906407);
+    path_0.cubicTo(size.width * 0.2726886, size.height * 0.1085936, size.width * 0.3839682, size.height * 0.06250000,
+        size.width * 0.5000000, size.height * 0.06250000);
+    path_0.cubicTo(size.width * 0.6160318, size.height * 0.06250000, size.width * 0.7273114, size.height * 0.1085936,
+        size.width * 0.8093591, size.height * 0.1906407);
+    path_0.cubicTo(size.width * 0.8914068, size.height * 0.2726886, size.width * 0.9375000, size.height * 0.3839682,
+        size.width * 0.9375000, size.height * 0.5000000);
+    path_0.cubicTo(size.width * 0.9375000, size.height * 0.6160318, size.width * 0.8914068, size.height * 0.7273114,
+        size.width * 0.8093591, size.height * 0.8093591);
+    path_0.cubicTo(size.width * 0.7273114, size.height * 0.8914068, size.width * 0.6160318, size.height * 0.9375000,
+        size.width * 0.5000000, size.height * 0.9375000);
+    path_0.close();
+    path_0.moveTo(size.width * 0.5000000, size.height);
+    path_0.cubicTo(size.width * 0.6326091, size.height, size.width * 0.7597841, size.height * 0.9473227,
+        size.width * 0.8535545, size.height * 0.8535545);
+    path_0.cubicTo(size.width * 0.9473227, size.height * 0.7597841, size.width, size.height * 0.6326091, size.width,
+        size.height * 0.5000000);
+    path_0.cubicTo(size.width, size.height * 0.3673909, size.width * 0.9473227, size.height * 0.2402159,
+        size.width * 0.8535545, size.height * 0.1464466);
+    path_0.cubicTo(
+        size.width * 0.7597841, size.height * 0.05267841, size.width * 0.6326091, 0, size.width * 0.5000000, 0);
+    path_0.cubicTo(size.width * 0.3673909, 0, size.width * 0.2402159, size.height * 0.05267841, size.width * 0.1464466,
+        size.height * 0.1464466);
+    path_0.cubicTo(
+        size.width * 0.05267841, size.height * 0.2402159, 0, size.height * 0.3673909, 0, size.height * 0.5000000);
+    path_0.cubicTo(0, size.height * 0.6326091, size.width * 0.05267841, size.height * 0.7597841, size.width * 0.1464466,
+        size.height * 0.8535545);
+    path_0.cubicTo(size.width * 0.2402159, size.height * 0.9473227, size.width * 0.3673909, size.height,
+        size.width * 0.5000000, size.height);
+    path_0.close();
+
+    final Paint paint0Fill = Paint()..style = PaintingStyle.fill;
+    paint0Fill.color = Colors.white.withOpacity(1.0);
```

Comment:
AppColors can be used here

Reply:
@gguijarro-c-chwy should we use always app colors instead defualt Colors even for colors like White ?

Reply:
yes, always app colors